projects/module-1/hello-worlds.md

```diff
@@ -0,0 +1,261 @@
+
+    - If you don't want to wait, your GH pages URL will follow this format: "https://username.github.io/hello-worlds/". Replace "username" with the repo owner's GH username and "hello-worlds" with your repo's name (if different). The URL will not work right away - that's ok! Add it to the Project Submission form anyway. 
+</section>
+
+<section class="answer">
+### Why are our PRs pushing to the Turing repo?
+
+Forks are typically used to let someone propose changes to your project, that would eventually be merged back into the original repo you forked from. So a fork will always remain connected to the original repo.
+
+When merging a PR, you should see a dropdown to select which repo you want to merge to. You can select the original repo or your fork (you want to choose your fork). 
+
+If you want to change the default, follow the steps [here](https://stackoverflow.com/questions/44466618/default-branch-for-pull-request).
+</section>
+
+<section class="answer">
+### Where are our console logs?
+
+If you console log something in your project, you’ll see it in the dev tools console when you open the page in your browser (`open index.html`). Nothing will appear in your terminal. Remember, you need to refresh the browser each time you make an update to your code.
+</section>
+</section>
+
+<section class="answer">
+### Refactoring as You Go
+
+Between each iteration, take some time to look at the code you've written and make any changes that might improve the readability of your code. Refactoring as you go can help ensure that everyone understands the code you've written so far AND make it easier to build on top of that code.
+
+Here are some ideas of what to look for:
+- Is our indentation and spacing consistent?
+- Are our variable names descriptive of the values they hold? Do they follow common convention (i.e. arrays are plural, etc)?
+- Are our function names descriptive of what the function does? Are they named with present tense verbs?
+- Are any of our functions too big or doing too many things? Should we break any of them up?
+- Is any of our JS code too repetitive? Could we make any reusable functions with parameters and arguments?
+
+Remember - shorter code isn't always better! 
+</section>
+
+## Progression
+
+<section class="answer">
+### Set Up + Day 1 Deliverables
+
+To begin, choose **ONE** partner to do the following:
+
+1. Fork the repository found here: [https://github.com/turingschool-examples/hello-worlds-boilerplate/](https://github.com/turingschool-examples/hello-worlds-boilerplate/).
+2. Clone down your new, forked repo.  While cloning, name it what you want your project to be named, should not include "boilerplate". `git clone <url> <newNameYouWantItToHave>`
+3. `cd` into the repository
+4. Open it in your text editor
+5. View the project in the browser by running `open index.html` in your terminal
+6. Add all project partners and your assigned instructor as collaborators on the repository
+
+Then, as a team:
```

Comment:
Do you want to add a note here that everyone else should clone down the repo at this point?

Reply:
Yes, great note! I'll add it.